### PR TITLE
create zh_Hans symlink, Fix #3668

### LINKED ISSUE #3668: Language code is not correct for Chinese
#### Please refer to the link below.

[https://code.djangoproject.com/ticket/18419](https://code.djangoproject.com/ticket/18419)

> This is because Traditional Chinese is not used only by Taiwan (tw) but also Honk Kong (hk), and Simplified Chinese is not only used in China (cn) but also Singapore (sg) and Malaysia. A new standard is to use "zh_Hant" for Traditional Chinese and "zh_Hans" for Simplified Chinese to eliminate the confusion of the content is not just for people lived in China or Taiwan.

### PR DESCRIPTION
**Pull request by yihuang**
Thanks for contributing to Wagtail! 🎉

Before submitting, please review the contributor guidelines <http://docs.wagtail.io/en/latest/contributing/index.html> and check the following:

* Do the tests still pass? (http://docs.wagtail.io/en/latest/contributing/developing.html#testing)
* Does the code comply with the style guide? (Run `make lint` from the Wagtail root)
* For Python changes: Have you added tests to cover the new/fixed behaviour?
* For front-end changes: Did you test on all of Wagtail’s supported browsers? **Please list the exact versions you tested**.
* For new features: Has the documentation been updated accordingly?

### CODE REVIEW

File: wagtail/api/v2/endpoints.py [hunk 1]
```diff
@@ -294,10 +295,7 @@ def get_serializer_class(self):
         request = self.request
 
         # Get model
-        if self.action == 'listing_view':
-            model = self.get_queryset().model
-        else:
-            model = type(self.get_object())
+        model = self.get_queryset().model
```
Comment (yihuang) [inline wagtail/api/v2/endpoints.py]: We can't call ``get_object`` when generates API schema. And I think there is no need to call ``get_object`` here anyway.

Comment (yihuang) [reply wagtail/api/v2/endpoints.py]: But I think the model of ``get_object`` should be the same with model of queryset, isn't it?

**Comment by yihuang** [comment]
I've created separate pull requests https://github.com/wagtail/wagtail/pull/4921 and https://github.com/wagtail/wagtail/pull/4922.

**Comment by gasman** [comment]
Migrating to the new names `zh_Hans` and `zh_Hant` was completed in d5bc3c92d22672f06f2aeab1e1731ca4f4fcb383.

**Pull request closed without merging**